src/std-types/exercise.md

```diff
@@ -0,0 +1,56 @@
+---
+minutes: 5
+---
+
+# Exercise: Hash Set
+
+In this exercise you will build a very simple hash set that stores `u32` values.
+The hash set will have a fixed size, and buckets should be selected with a
+simple modulus operation (`i % num_buckets`). Use a `Vec` to represent each
+bucket (separate chaining). Hint: a vector can contain other vectors:
+`Vec<Vec<u32>>`.
```

Comment:
I don't think this is quite the right level of exercise to test/develop knowledge of the stdlib. We haven't really explained generics, ownership, or borrowing yet, and students coming from high-level languages may not be familiar with terminology for hash table implementations like "buckets" and "chaining".

What specific takeaways do we want students to derive from this exercise?